setup.py

```diff
@@ -21,8 +21,11 @@
       description='Distances and representations of persistence diagrams',
       long_description=long_description,
       long_description_content_type="text/markdown",	
-      author='Nathaniel Saul, Chris Tralie, Francis Motta',
-      author_email='nat@riverasaul.com, chris.tralie@gmail.com, francis.c.motta@gmail.com',
+      author='Nathaniel Saul, Chris Tralie, Francis Motta, Michael Catanzaro,
+      Gabrielle Angeloro, Calder Sheagren',
+      author_email='nat@riverasaul.com, chris.tralie@gmail.com,
```

Comment:
I looks like there's newlines between names and emails. I dont think that will work correctly with python strings.  What happens when you reinstall it with `pip install -e .`?

Reply:
Yes, I'll fix this soon. Thanks for catching it.